### ISSUE TITLE
Using in Cloudflare Worker with PKCS#8 key results in `ERR_OSSL_ASN1_HEADER_TOO_LONG`

### ISSUE THREAD

**Issue by kylebjordahl** [gr2m/universal-github-app-jwt#70]
I'm building a little deployment dashboard for GitHub using Remix and Cloudflare Pages. I've got simple OAuth interactions working using the `octokit` implementation.

When I went to use app-based authentication, I got the expected console error about `#1` vs `#8` encoding, so I fixed that and now I'm getting the following error: 

```
Error: error:0D07207B:asn1 encoding routines:ASN1_get_object:header too long
    at createPrivateKey (node:internal/crypto/keys:620:12)
    at Object.rsaImportKey (node:internal/crypto/rsa:258:19)
    at SubtleCrypto.importKey (node:internal/crypto/webcrypto:536:10)
    at Proxy.importKey (__REDACTED__/node_modules/@miniflare/core/src/standards/crypto.ts:162:38) {
  library: 'asn1 encoding routines',
  function: 'ASN1_get_object',
  reason: 'header too long',
  code: 'ERR_OSSL_ASN1_HEADER_TOO_LONG'
}
```

I seem to get the same error when running locally in miniflare/worker dev and deployed.

I'm _inclined_ to think that this could be a Cloudflare implementation issue, but I was hoping you might be able to confirm that before I pull too much more hair out.

**Comment by gr2m** [comment]
I'm sorry I don't know

**Comment by kylebjordahl** [comment]
All good, thanks for the response!

**Comment by kylebjordahl** [comment]
I figured it out! While online key checkers such as [https://lapo.it/asn1js/](https://lapo.it/asn1js/) correctly parse a PKCS#8 key where there is no line break between the header and the body, something about the particular implementation here cares, causing it to parse incorrectly. And of course, I'm storing the key as a single line because it needs to be passed in via the Cloudflare worker ENV system for local development.

I solved this locally by using the following: 
```typescript
const formattedKey = context.GITHUB_PRIVATE_KEY.replace(
        "----BEGIN PRIVATE KEY-----",
        "----BEGIN PRIVATE KEY-----\n"
).replace(
        "-----END PRIVATE KEY-----", 
        "\n-----END PRIVATE KEY-----"
);
```
